### PR TITLE
Stronghold Content Update

### PR DESCRIPTION
**Pull request by fernandogelin**
### Pull Request
---

#### PR Summary
Briefly describe the changes.

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [ ] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [x] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [x] Commitizen was used for all commits.
- [x] Local site looks as expected after changes.
- [x] Contributing guidelines were followed.
- [x] If changes affect development, was the README updated?

##### If PR to `production`
- [x] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

**Comment by fernandogelin** [comment]
@brown-ccv/website-admin DO NOT MERGE UNTIL #178 is merged into `master`

**Comment by mcmcgrath13** [comment]
To match the convention for ARC, Yang should be before Sign in the systems group - assuming they be divided into distinct teams now, which maybe is not true? This is perhaps more of a new issue than related to this PR. Also, Raj and Sam need to be removed ☹️ .

File: data/opportunities.yaml [hunk 1]
```diff
@@ -2,18 +2,14 @@
   team: CCV Operations
   subteam: User Services
   link: "https://brown.wd5.myworkdayjobs.com/en-US/staff-careers-brown/job/180-George-Street/Research-Services-Support-Specialist_REQ151702"
-- title: Manager CCV Service Center
-  team: CCV Operations
-  subteam: User Services
-  link: "https://brown.wd5.myworkdayjobs.com/en-US/staff-careers-brown/job/180-George-Street/Manager-CCV-Service-Center_REQ161111"
 - title: Research Software Engineer/Senior Research Software Engineer
```
Comment (mcmcgrath13) [inline data/opportunities.yaml:5]: This link seems to be dead

**Review by mcmcgrath13 — COMMENTED** [review]
One of the HPC job links is dead

**Review by mcmcgrath13 — APPROVED** [review]
The issues I found are existing bugs, and not newly introduced in this PR.  So I approve, and those can be addressed separately.

**Pull request merged**